### PR TITLE
Fix bug hire and reject commands don't update status immediately

### PR DESCRIPTION
**Pull request by bigismols**

### CODE REVIEW

**Review by SQ77 — COMMENTED** [review]
When no person is selected (clicked):
shows updated status immediately for people with "hired" or "rejected" tags, not those with "pending" tags

When a person is selected (clicked):
does not show updated status immediately for all people

**Comment by bigismols** [comment]
Fixes #106

File: src/main/java/seedu/address/logic/commands/HireCommand.java [hunk 1]
```diff
@@ -55,9 +56,12 @@ public CommandResult execute(Model model) throws CommandException {
...
         }
 
-        personToHire.markAsHired();
-        return new CommandResult(String.format(MESSAGE_SUCCESS,
-                personToHire.getName()));
+        Person personHired = new Person(personToHire.getName(), personToHire.getJob(), personToHire.getPhone(),
+                personToHire.getEmail(), personToHire.getTags());
+        personHired.markAsHired();
+        model.setPerson(personToHire, personHired);
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```
Comment (LemonDrew) [inline src/main/java/seedu/address/logic/commands/HireCommand.java:63]: Just to clarify, this updates the list before returning?

**Review by LemonDrew — APPROVED** [review]
LGTM!

**Review by SQ77 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/HireCommand.java (hunk 1 above)
Comment (bigismols) [reply src/main/java/seedu/address/logic/commands/HireCommand.java:63]: yessir

**Pull request merged**